Source/UI/NewUI/BillingForm/View/BillingFormTextField.swift

```diff
@@ -6,6 +6,7 @@ class BillingFormTextField: UITextField {
     init(type: BillingFormCell?, tag: Int) {
         self.type = type
         super.init(frame: .zero)
+        backgroundColor = .clear
```

Comment:
This is nil by default, so I am not sure we ever need to set it to clear?

Reply:
does `nil` means same as `clear` ? if then makes sense, otherwise can stay `clear` does not harm. I think the overall output will be same provided one line of extra code which can be removed.

Reply:
I have tested removing this line then background is `.white`. For now, Gonna keep it `.clear`